src/initStackArgs.ts

```diff
@@ -7,11 +7,16 @@ export async function initStackArgs(argv: Arguments): Promise<number> {
     Type: "AWS::CloudFormation::WaitConditionHandle"
     Properties: {}`;
 
-  const stackArgs = `# REQUIRED SETTINGS:
+  const stackArgs = `# INITIALIZED STACK ARGS
+# $imports:
+#   environment: env:ENVIRONMENT
```

Comment:
not sure if we need this since we already have `Tags: environment:`. Maybe we can use that?